agent/ec2_meta_data_test.go

```diff
@@ -0,0 +1,28 @@
+package agent
```

Comment:
Open to ideas for how to test this — ping @lox

Reply:
I'll 🧠 today.

Reply:
@lox I’m sure it’s pretty easy with a little test http server but I ran out of gojuice to work it out